npm-binary-distributions/darwin/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@sentry/cli-darwin",
```

Comment:
Do we need to have a separate binary for M1/M2 and Intel Macs?

Reply:
The CLI is currently set up to use our universal darwin binary no matter what architecture.

https://github.com/getsentry/sentry-cli/blob/1c5d20f94043ac66631157b2a208d5668541921c/scripts/install.js#L78-L80

Also, I believe mac OS has a compatibility layer for non-arm binaries. But good that you confirmed!

Reply:
Yeah, but I believe the Rosetta compatibility layer you are referencing imposes a performance penalty, and also, if it is the first time the user is running a non-arm binary, the user is prompted to install Rosetta and needs to wait some time for the install to complete. These are probably only minor inconveniences, but we should probably be creating a native ARM build (as we already are with the universal binary)